pyscf/adc/uadc.py

```diff
@@ -585,11 +581,27 @@ def compute_amplitudes(myadc, eris):
 
         t1_3 = (t1_3_a, t1_3_b)
 
+    del (D1_a, D1_b) 
+
+    if not isinstance(eris.oooo, np.ndarray):
+        t2_1_a = radc_ao2mo.write_dataset(t2_1_a)
```

Comment:
You don't to store all three spin cases of each amplitude array (e.g., a, ab, b) at the same time. Once one spin case is computed, you should dump it to disk to free memory. Please apply this to all spin cases of double-excitation amplitudes.

Reply:
Done